src/main.ts

```diff
@@ -110,7 +121,6 @@ async function run(): Promise<void> {
     // Check Tailscale status (cross-platform)
     try {
       const status = await getTailscaleStatus();
-      core.debug(`Tailscale status: ${JSON.stringify(status)}`);
```

Comment:
This is redundant with `stdout` and less readable because it's been round-tripped and isn't pretty-printed.